core/state_transition.go

```diff
@@ -304,6 +304,12 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 	// 5. there is no overflow when calculating intrinsic gas
 	// 6. caller has enough balance to cover asset transfer for **topmost** call
 
+	// If ETX, bypass accessList enforcement
+	if st.msg.IsETX() {
+		orig := st.state.ConfigureAccessListChecks(false)
+		defer st.state.ConfigureAccessListChecks(orig)
```

Comment:
dont understand the use of this defer, can you explain?

Reply:
is it to revert the change just made for the execution of the etx?

Reply:
yes, exactly. I set the bypass option temporarily for that call, but want to revert it as soon as the call is complete.